homeassistant/components/huawei_lte/sensor.py

```diff
@@ -36,53 +37,66 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-SENSOR_META = {
-    KEY_DEVICE_INFORMATION: dict(
+class SensorMeta(NamedTuple):
```

Comment:
Python 3.7 supports dataclasses.

https://docs.python.org/3/library/dataclasses.html

Reply:
Sure, but to me NamedTuple is a better fit as the data is intentionally immutable, and none of the additional functionality offered by dataclasses over NamedTuples is needed here.

Reply:
Immutability can be emulated with the `frozen` parameter of dataclasses.

https://docs.python.org/3/library/dataclasses.html#frozen-instances

Reply:
Yes, but we can just use NamedTuple as it provides spot on what we want here.